`token_sequence` must be incremented in `validator-keys.json` backups (#878)
XRPLF/xrpl-dev-portal: content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

        This file should be stored securely and not shared.

>       **Warning:** Store the generated `validator-keys.json` key file in a secure, offline, and recoverable location, such as an encrypted USB flash drive. Do not store keys on the validator where you intend to use the keys. If your `secret_key` is compromised, [revoke the key](https://github.com/ripple/validator-keys-tool/blob/master/doc/validator-keys-tool-guide.md#key-revocation) immediately. Do not modify the contents of `validator-keys.json`, except to increment the `token _sequence` after generating a new validation token using a copy of the file. For example, if you copy your secure, backed up `validation-keys.json` to your validator to generate a new token, generating the token will increase the `token_sequence` field. If you do not update the `token_sequence` field in your secure backup, then the next token you generate will have the same or lower sequence number as the token you are currently using. The network will ignore validations that have a sequence number that is less than the previously highest observed number. Alternately, simply copy `validator-keys.json` to your backup location every time you generate a new validation token, so your backup version has the correct `token_sequence`.

Reviewer: 1. You should not need to copy your `validation-keys.json` file to your validator. You should, ideally, run the validator keys tool on a separate machine to update the keys.

2. I don't want to instruct people to update the `token_sequence` manually except in special cases, because I'd like these instructions to be future-proof against changes in the validator keys tool. For example, @nbougalis was thinking tokens could switch to a time-based instead of sequence-based system for marking new tokens. There's probably a separate place (troubleshooting?) to cover the [special case](https://github.com/ripple/xrpl-dev-portal/issues/879).

2. I'm suggesting a couple edits to remove some words like [simply](https://justsimply.dev/) as a best practice.

```suggestion
      **Warning:** Store the generated `validator-keys.json` key file in a secure, offline, and recoverable location, such as an encrypted USB flash drive. Do not store keys on the validator where you intend to use the keys. If your `secret_key` is compromised, [revoke the key](https://github.com/ripple/validator-keys-tool/blob/master/doc/validator-keys-tool-guide.md#key-revocation) immediately. Do not modify the contents of `validator-keys.json`, except to update the backup after generating a new token. If you generate more than one token from the same backup without updating, the network ignores the later tokens because they use the same `token_sequence` number.
```

Author: All three points make sense, and I like how much more concisely you get the message across.
